tests/smoke/.makim-scheduler.yaml

```diff
@@ -17,19 +35,19 @@ groups:
       test-all:
         hooks:
           pre-run:
-            - task: test-echo
-            - task: test-date
-            - task: test-sleep
+            - task: test.test-echo
```

Comment:
good catch, the tasks should have the "test" group. could you open a PR with this change?